src/univalue/include/univalue.h

```diff
@@ -49,12 +46,12 @@ class UniValue {
 
     void setNull();
     void setBool(bool val);
-    void setNumStr(const std::string& val);
+    void setNumStr(std::string str);
     void setInt(uint64_t val);
     void setInt(int64_t val);
     void setInt(int val_) { return setInt(int64_t{val_}); }
     void setFloat(double val);
-    void setStr(const std::string& val);
+    void setStr(std::string str);
```

Comment:
In commit "univalue: Avoid std::string copies" (fa1ce3ab42fd5fd761b934b8db9fbf9f78bde842)

Not sure if this was intended, but this is breaking the pattern of calling all `setXXX` arguments `val`

Reply:
Yes this is intended. Note that the argument are named inconsistently `val`, and `val_`. Happy to change to something other than `str`, maybe `val_in`? Though, I think the args should be named the same in the header file as they are in the cpp file.